data/pug.txt

```diff
@@ -317,6 +317,16 @@ outfit "Pug Biodefenses"
 	"unplunderable" 1
 	description "This defense system is designed to release a suite of toxins that are lethal to human beings (and probably to some other species, as well) but which have no effect at all on the Pug."
 
+outfit "Pug Peacekeeping Staff"
+	plural "Pug Peacekeeping Staves"
```

Comment:
"Staffs" is a more common plural form of staff, so no need for this.

Reply:
Is it though? Staffs is also a verb, so comparing word usage in general isn't straightforward. When talking about a beat-you-on-the-head stick, "staves" sounds better to me than "staffs." Plus, I think "staves" is better-suited to the Pug.

Reply:
I agree. Whenever there are multiples in books (fantasy and/or medieval genre), they generally use staves in my experience.